### PR TITLE
Added Column Chart

### PR DESCRIPTION
**Pull request by desafinadude**
Story showing:
- Highlight Fix 
- SVG filter fix

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,13870 @@
+{
```
Comment (jbothma) [inline package-lock.json]: we're using yarn now, not npm, so you'll need to update yarn.lock instead

**Comment by jbothma** [comment]
The chart positioning has regressed a bit - now it has no connection to the bar in the group that's being highlighted.

Screenshot_2020-12-08_14-04-35

**Comment by jbothma** [comment]
How should I verify the highlight fix?

If I

1. Click generate
2. click highlight - see second series cape town highlighted
3. click swap data
4. click highlight

nothing is highlighted

**Comment by desafinadude** [comment]
> How should I verify the highlight fix?
> 
> If I
> 
> 1. Click generate
> 2. click highlight - see second series cape town highlighted
> 3. click swap data
> 4. click highlight
> 
> nothing is highlighted

The second series doesn't have CPT so nothing should be highlighted then.
I will add more data to illustrate

File: package-lock.json (hunk 1 above)
Comment (desafinadude) [reply package-lock.json]: Switched to yarn

**Comment by desafinadude** [comment]
> The chart positioning has regressed a bit - now it has no connection to the bar in the group that's being highlighted.
> 
> Screenshot_2020-12-08_14-04-35

Fixed alignment

**Comment by desafinadude** [comment]
1. Generate a chart (this is just a storybook specific issue - the chart does not generate a container by itself - and the container is not present in the DOM at that time)
2. Load Data - this will load an intital dummy comparison set
3. Highlight - this will highlight CPT
4. Swap Data - this will load new data that doesn't include CPT
5. Highlight - this will do nothing as CPT does not exist in the set...so highlighting is no longer confused

**Comment by jbothma** [comment]
But the problem was that I want to see something get highlighted when different municipalities are used in the comparison. Perhaps another highlight button that demonstrates that one of the municipalities in the second set _can_ be highlighted?

Perhaps keep the current button as is so we _can_ see that cape town isn't highlighted. But I also want to see that a new comparison can.

**Comment by desafinadude** [comment]
> But the problem was that I want to see something get highlighted when different municipalities are used in the comparison. Perhaps another highlight button that demonstrates that one of the municipalities in the second set _can_ be highlighted?
> 
> Perhaps keep the current button as is so we _can_ see that cape town isn't highlighted. But I also want to see that a new comparison can.

Added a second highlight button for the second series.

**Pull request merged**